src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -245,7 +245,8 @@ export default class ClinicalData extends React.Component<IClinicalDataProps, {}
     private readonly groupSampleDataPromise = remoteData({
         await: () => [this.props.store.sampleKeyToActiveGroups, this.props.store.uidToGroup],
         invoke: async () => {
-            const axisData: IAxisData = { data: [], datatype: "string" };
+            const categoryOrder = _.map(this.props.store.uidToGroup.result!,group=>group.nameWithOrdinal).sort()
```

Comment:
This is hacky to me - ordinals being alphabetical order isn't a guarantee of the ordinals even though it sort of makes sense. It's just logical wire-crossing IMO. Better to do it official way which is based on the order in _originalGroups. `stringListToIndexSet` is useful for this